### ISSUE TITLE
Connect to online server

### ISSUE THREAD

**Issue by John-Naiker** [arpruss/raspberryjammod#61]
Hey. This is amazing! How would one initiate a connection from Scratch or Python to a minecraft server running online rather than locally?

**Comment by Marcinosoft** [comment]
Yes, but using "RaspberryJuice" plugin dedicated to Minecraft servers like Bukkit, Spigot or Paper. Plugin was developed for Minecraft Java Edition 1.12.1, but It works with latest one 1.18.1 too. This plugin allow to connect to server running on localhost, LAN or via Internet.

https://github.com/zhuowei/RaspberryJuice

**Comment by xboxone08** [comment]
If you can install plugins on the server, you could install [RaspberryJuice](https://github.com/zhuowei/RaspberryJuice) (Bukkit/Spigot/Paper plugin) on it and then be able to control it using Python. You can't control a server which doesn't use the plugin with Python though.
